Update 04-hosszukard.md

gomolygo opened the pull request:
kibaszott wall of text xdddd

gomolygo committed:
Update 04-hosszukard.md

gomolygo committed:
Create 04-hosszukard.md

Csel hatarozott megakasztasa
Formazasi hiba
1/2 1/3 helyett
Fokél faszsag az elejen
Védésnel mindket tipusra vonatkozzon a reszletezes
Kitéréssel védés
Pontos szog helyett tompa szog
Vor torolve

gomolygo committed:
Update 04-hosszukard.md

- heje sírás ellenörzés ala google
- két vor bent maradt

SenkiAlphonse commented on 04-hosszukard.md:
```diff
@@ -1,70 +1,106 @@
+ 
+3. Pengén keresztül - amelyek egyidejüleg találják el a test érvényes találati felületét és az ellenfél fegyverét- érkező találatok érvényesek minden olyan alkalommal, amikor tisztán eltalálják az érvényes találati felületet.
+ 
+4. A markolatgombbal végrehajtott találatok érvényesnek, ha az ellenfél maszkjának borítás nélküli rostélyát találják el jelzés értékűen -akár a gombot kesztyűvel eltakarva- és biztonságosan kivitelezve. A szükségtelen erővel végrehajtott markolatgomb akciók a szabálytalanságok második csoportjában foglaltak szerint büntetendő.
```
typo: 'érvényesek'

SenkiAlphonse replied:
vagy 'érvényesnek számítanak, '

SenkiAlphonse commented on 04-hosszukard.md:
```diff
@@ -1,70 +1,106 @@
+ 
+1. Csak azok a találatok érvényesek, amelyek az érvényes találati felületre esnek.
+ 
+2. Találati felület a vívó teljes teste, beleértve a fegyver markolatát, a ruházatot és a védőfelszerelést, kivéve a fej hátulja, a gerincfe, a lábfejek és a térdek hátsó része.
```
typo: 'gerinc' -'fe'

SenkiAlphonse commented on 04-hosszukard.md:
```diff
@@ -1,70 +1,106 @@
+ 
+2. Találati felület a vívó teljes teste, beleértve a fegyver markolatát, a ruházatot és a védőfelszerelést, kivéve a fej hátulja, a gerincfe, a lábfejek és a térdek hátsó része.
+ 
+3. Ha egy találat az érvényes találati felületen kívülre érkezik és nem számít találatnak.
```
Ez kicsit redundáns a 15. után. Esetleg így: Az érvényes találati felületen kívül eső találatok érvénytelenek, az akciók megítélése során nem vehetők figyelembe.

SenkiAlphonse replied:
esetleg mehetne a találatok megállapítása vagy érvénytelenítése alá

SenkiAlphonse commented on 04-hosszukard.md:
```diff
@@ -1,70 +1,106 @@
+ 
+1. A találatok elbírálásának alapját a mérkőzést vezető, az akciót megfigyelő bíró és segédje észlelései képezik.
+ 
+2. Minden szabálytalanság érvényteleníti a vétkes vívó által adott találatot.
```
így? : Minden szabálytalanság érvényteleníti a vétkes vívó által adott találatot, amely során a szabálytalanságot elkövette.

(ld. pl a bíró nem álllítja meg az akciót egy szabálytalanságnál, amiből nem esik találat. Ezután az elkövető talál - itt nem tudom mennyire érdemes érvényteleníteni, fie nem szokta, ld a 2013 hun-kor csapat döntő. Az is jó, ha másként csináljuk, de ezt sztem édemes egyértelműen beleírni)

SenkiAlphonse commented on 04-hosszukard.md:
```diff
@@ -1,70 +1,106 @@
+ 
+2.  Minden helyesen végrehajtott támadást védeni kell vagy teljesen el kell kerülni és vívómenetet szabályosan végrehajtva kell folytatni.
+ 
+3.  Támadó lábmunkának számít a kitörés, lépés kitörés, lerohanás (flèche) és dinamikus kereszt lépés. Az előre haladás és lépés előkészítésnek minősül és ezzel az előkészítéssel szemben minden egyszerű támadásnak elsőbbsége van.
```
az 'előre haladás'-t kivenném, mert önmagában jelenthet támadásnak minősülő dolgokat is. 

itt megtartanám a régit, mert pontosabb:

'A behajlított karral végrehajtott akciók, akár egyszerű, akár összetett, lépés vagy csel, nem jelentenek közvetlen fenyegetést, és nem minősülnek támadásnak, csak előkészítésnek, amelyek ki vannak téve az ellenfél által kezdeményezett támadásnak vagy közbetámadásnak. Az ilyen akciók közben elszenvedett támadásokból, közbetámadásokból származó találatok érvényesnek számítanak, és pontot érnek az előkészítésen ért vívó ellen.'

SenkiAlphonse replied:
sőt most megtaláltam hogy később ott van ez isa 49. sorban. Lehet h elég lenne egy referenciát írni ide, hogy a támadást nem indító vívó mozgolódására mi vonatkozik

SenkiAlphonse commented on 04-hosszukard.md:
```diff
@@ -1,70 +1,106 @@
+ 
+3.  Támadó lábmunkának számít a kitörés, lépés kitörés, lerohanás (flèche) és dinamikus kereszt lépés. Az előre haladás és lépés előkészítésnek minősül és ezzel az előkészítéssel szemben minden egyszerű támadásnak elsőbbsége van.
+ 
+4. A támadás akkor van szabályosan végrehajtva, ha a kar folyamatos nyújtása közben, a penge hegye vagy éle folyamatosan fenyegeti az érvényes találati felületet, megelőzve a támadó lábmunka megindítását.
```
a 'progresszíven' sztem kell ide, mert az mondja azt h folyamatosan nyújtani kell a kart, tehát nem elég csak rámutatani és kitörni, a kéznek mennie kell előre. Itt is tartanám a régit, sztem elfér:

'Az támadás kivitelezése akkor helyes, ha annak során a fegyver hegye vagy éle folyamatosan és progresszíven fenyegeti az ellenfél érvényes találati felületét, tehát a kar kinyújtásának megkezdése megelőzi a támadó lábmunka indító mozdulatát."

gomolygo replied:
progresszíven - azért tartom túlmagyarázásnak vagy túl komplikálásnak mert a helyes támadásnál a kéz előbb indul mint a láb, de az a FIE szabályzatban sem elvárás hogy pont abban a pillanatban legyen 100%nyújtva mondjuk egy szúrásnál amikor célba ér: 
az is helyes támadás ha én megindítom a kezem, utána a lábam, a levegőben már célba érés előtt telejsen kinyúlik a kezem és csak utána érkezik be

SenkiAlphonse replied:
arra talán érdemes lenn utalni, h ha közben visszatartod vagy visszahúzod a kezed, akkor pl ha a másik jól támad, ő fog nyerni akkor is ha egyszerre indultatok. Erre volt ez a progresszív, de nekem más megoldás is jó.

SenkiAlphonse commented on 04-hosszukard.md:
```diff
@@ -1,70 +1,106 @@
+ 
+5. A támadások helyességének elbírálásakor a következőket kell figyelembe venni:
+   1. Az egyszerű támadás, akár közvetlen, akár közvetett, helyesen végrehajtott akkor, ha a kar nyújtása az érvényes találati felületet fenyegető heggyel vagy éllel megelőzi a támadó lábmunka megindítását.
+   2. Az összetett támadás, helyesen végrehajtott akkor, amikor a kar kinyújtásának kezdete az első csel
```
typo: támadás után nem kell vessző a 46. sorban

SenkiAlphonse commented on 04-hosszukard.md:
```diff
@@ -1,70 +1,106 @@
+5. A támadások helyességének elbírálásakor a következőket kell figyelembe venni:
+   1. Az egyszerű támadás, akár közvetlen, akár közvetett, helyesen végrehajtott akkor, ha a kar nyújtása az érvényes találati felületet fenyegető heggyel vagy éllel megelőzi a támadó lábmunka megindítását.
+   2. Az összetett támadás, helyesen végrehajtott akkor, amikor a kar kinyújtásának kezdete az első csel
+   helyes bemutatásánál megelőzi a támadó lábmunka megindítását és a találat legkésőbb annak befejezésekor beérkezik.
```
azt, h 'helyes bemutatás' nem írnám bele, szerintem elég annyi h csel. Ha valamit rosszul mutatnak be, azt nem nevezzük cselnek oszt kész. (a magyarázkodás kiirtásának szellemében)

SenkiAlphonse commented on 04-hosszukard.md:
```diff
@@ -1,70 +1,106 @@
+ 
+7. A támadás elsőbbségének megítéléséhez az alábbi tényezőket kell figyelembe venni:
+   1. Ha a támadás akkor indul, amikor az ellenfél vonalban van (azaz nyújtott karral, az érvényes találati felületet fenyegető heggyel - hosszúhegy/langort), akkor a támadónak először ki kell térítenie az ellenfél fegyverét. A mérkőzést vezető bírónak figyelnie kell arra, hogy a pengék puszta érintkezése nem tekinthető elégségesnek az ellenfél fegyverének kitérítésére.
+   2. Ha az eltérítést célzó penge keresés sikertelen, az akció joga átkerül az ellenfélhez.
```
typo penge-keresés (asszem. Vagy egyben, Külön fura.)

SenkiAlphonse commented on 04-hosszukard.md:
```diff
@@ -1,70 +1,106 @@
+ 
+8. Összetett támadások, cselek helyességének elbírálásakor az alábbi tényezőket kell figyelembe venni:
+   1. Az összetett támadásoknál a cseleket helyesen kell bemutatni, azaz:
+       1. a szúrócselt a kar nyújtásával, miközben a hegy állandóan fenyegeti az érvényes találati felületet;
```
itt 'a kar nyúltás' dolgot részletezném, mert ugye ha csak kinyújtja a karját az már nem csel stb. 
Valami olyasmivel, hogy az összetett támadás helyesen végrehajtottnak tekintendő, ha a mozdulatokat (beleértve a cseleket, pengetámadásokat, stb) a kar progresszív nyújtásával hajtják végre, a kar visszahúzása nélkül, miközben a fegyver hegye vagy éle folyamatosan fenyegeti az ellenfél érvényes találati felületét.

SenkiAlphonse commented on 04-hosszukard.md:
```diff
@@ -1,70 +1,106 @@
+   1. Az összetett támadásoknál a cseleket helyesen kell bemutatni, azaz:
+       1. a szúrócselt a kar nyújtásával, miközben a hegy állandóan fenyegeti az érvényes találati felületet;
+       2. a vágócselt a kar nyújtásával, miközben a penge és a kar tompa szöget alkot és a penge éle állandóan fenyegeti az érvényes találati felületet.
+   2. Ha az ellenfél egy összetett támadásnál valamelyik csel közben elfogja a pengét és kötéssel megakadályozza annak eredeti útvonalát, jogot nyer a riposztra.
```
kötéssel vagy ütéssel eltéríti azt eredeti útvonalából?

SenkiAlphonse commented on 04-hosszukard.md:
```diff
@@ -1,70 +1,106 @@
+   2. Ha az ellenfél egy összetett támadásnál valamelyik csel közben elfogja a pengét és kötéssel megakadályozza annak eredeti útvonalát, jogot nyer a riposztra.
+   3. Összetett támadásnál az ellenfélnek joga van közbetámadást végrehajtani. Ahhoz azonban, hogy az érvényes legyen, a közbetámadásnak a támadás befejezését egy vívóidővel meg kell előznie, azaz a közbetámadásnak találnia kell, mielőtt a támadó megkezdte volna támadása befejezésének utolsó mozdulatát.
+ 
+9. Támadás pengeütéssel:
```
Pengeütéssel vagy -kötéssel

SenkiAlphonse commented on 04-hosszukard.md:
```diff
@@ -1,70 +1,106 @@
+   3. Összetett támadásnál az ellenfélnek joga van közbetámadást végrehajtani. Ahhoz azonban, hogy az érvényes legyen, a közbetámadásnak a támadás befejezését egy vívóidővel meg kell előznie, azaz a közbetámadásnak találnia kell, mielőtt a támadó megkezdte volna támadása befejezésének utolsó mozdulatát.
+ 
+9. Támadás pengeütéssel:
+   1. Pengeütéses támadásnál a támadás akkor tekinthető helyesen végrehajtottnak és akkor részesül elsőbbségben, amikor a pengeütés az ellenfél pengéjének gyengéjére, azaz a keresztvastól legtávolabb eső 1/2 részére esik.
```
A pengeütéssel vagy -kötéssel végrehajtott támadások akkor tekinthetők helyesen végrehajtottnak és akkor részesülnek elsőbbségben, ha a pengetámadás eltéríti az ellenfél fegyverét eredeti pályájáról, közvetlen lehetőséget adva a támadónak a találatszerzésre; valamint amikor a pengeütés az ellenfél pengéjének gyengéjére, azaz a keresztvastól legtávolabb eső 1/2 részére esik.

SenkiAlphonse commented on 04-hosszukard.md:
```diff
@@ -1,70 +1,106 @@
+ 
+9. Támadás pengeütéssel:
+   1. Pengeütéses támadásnál a támadás akkor tekinthető helyesen végrehajtottnak és akkor részesül elsőbbségben, amikor a pengeütés az ellenfél pengéjének gyengéjére, azaz a keresztvastól legtávolabb eső 1/2 részére esik.
+   2. Pengeütéses támadásnál, amikor a pengeütés az ellenfél pengéjének erősére esik, azaz a keresztvashoz legközelebb eső 1/2 részére üt, a támadást rosszul végrehajtottnak kell tekinteni és ez a pengeütés elsőbbséget ad az ellenfél azonnali riposztjának.
```
'pengeütéssel vagy -kötéssel végrehajtott támadás etc...'

SenkiAlphonse commented on 04-hosszukard.md:
```diff
@@ -1,70 +1,106 @@
+10. Védés
+   1. A megfelelően végrehajtott védés jogot ad a riposztra (visszatámadásra): az egyszerű visszavágás (-szúrás) lehet közvetlen vagy közvetett, azonban ahhoz, hogy a támadó minden további akcióját érvénytelenítse, azt azonnal kell végrehajtani határozatlanság vagy szünet nélkül.
+   2. Szúrások elleni védésnek az a célja, hogy az megakadályozza a hegy beérkezését a találati felületre, tehát a védés akkor helyes, ha az akció végén az ellenfél hegye nem fenyegeti a védőt.
+   3. A vágások elleni védésnek az a célja, hogy megakadályozza a találat beérkezését arr a találati felületre, amely ellen az ellenfél vágásai irányulnak és az ellenfél hegye nem veszélyeztetni a védőt.
```
typo: arra

SenkiAlphonse commented on 04-hosszukard.md:
```diff
@@ -1,70 +1,106 @@
+   1. A megfelelően végrehajtott védés jogot ad a riposztra (visszatámadásra): az egyszerű visszavágás (-szúrás) lehet közvetlen vagy közvetett, azonban ahhoz, hogy a támadó minden további akcióját érvénytelenítse, azt azonnal kell végrehajtani határozatlanság vagy szünet nélkül.
+   2. Szúrások elleni védésnek az a célja, hogy az megakadályozza a hegy beérkezését a találati felületre, tehát a védés akkor helyes, ha az akció végén az ellenfél hegye nem fenyegeti a védőt.
+   3. A vágások elleni védésnek az a célja, hogy megakadályozza a találat beérkezését arr a találati felületre, amely ellen az ellenfél vágásai irányulnak és az ellenfél hegye nem veszélyeztetni a védőt.
+   4. A védés végrehajtása akkor helyes, amikor a támadó akció befejezése előtt ellenáll az ellenfél találata beérkezésének, lezárva azt a vonalat, amelyben ennek a támadó akciónak be kell fejeződnie és az ellenfél pengéjének a hegyét kitéríti, hogy az ne veszélyeztessen érvényes találati felületet.
```
a mondat utolsó része: 
'és az ellenfél pengéjének a hegyét megakadályozza az érvényes találati felület veszáélyeztetésében' - hacsak nem szándékos, szerintem ez kicsit jobb mint a kitéríti, ami egy kifejezetten aktív ige (ami a pengetámadásoknál pl helyes), a megakadályozzába meg belefér az is h csak útját állja (ami parád esetén sztem pontosabb).

SenkiAlphonse commented on 04-hosszukard.md:
```diff
@@ -1,70 +1,106 @@
+   3. A vágások elleni védésnek az a célja, hogy megakadályozza a találat beérkezését arr a találati felületre, amely ellen az ellenfél vágásai irányulnak és az ellenfél hegye nem veszélyeztetni a védőt.
+   4. A védés végrehajtása akkor helyes, amikor a támadó akció befejezése előtt ellenáll az ellenfél találata beérkezésének, lezárva azt a vonalat, amelyben ennek a támadó akciónak be kell fejeződnie és az ellenfél pengéjének a hegyét kitéríti, hogy az ne veszélyeztessen érvényes találati felületet.
+   5. Ha egy védést helyesen hajtottak végre, az ellenfél támadó akcióját kivédettnek kell minősíteni és ítélni a mérkőzést vezető bírónak még akkor is, ha a támadó pengéjének hajlékonysága miatt annak vége eléri a célba vett felületet -például áthajlik a keresztvas felett a penge.
+   6. A támadás elkerülése védésnek számít és annak megfelelően ad riposzt jogot.
```
'A támadás elkerülésével (a találati felület találat előli kitérítésével) a védekező vívó a védéseknél leírtaknak megfelelően szerez jogot az azonnali riposztra.'

-így nem is írtuk le h az 'egy védés', de az eredmény ugyanaz

SenkiAlphonse commented on 04-hosszukard.md:
```diff
@@ -1,70 +1,106 @@
-6.  Ha mindkét vívó egy kézzel fogott a fegyverrel kap találatot egy tempón belül, a szokásos előnyszabályok érvényesülnek.
+1. Érvényes találatot olyan támadással lehet szerezni, amelynek a meginditásának pillanatában a vívó nem volt közvetlenül fenyegetve ellenfele által.
+ 
+2. Amikor egy vívómenetben mindkét vívó egyidejűleg találatot, az vagy együttes találat, vagy pedig kettős találat.
```
'Amikor egy vívómenetben mindkét vívó egyszerre talál, ... '

gomolygo committed:
Update 04-hosszukard.md

- PR comment resolve:
- érvényesek
-gerinc
- Ez kicsit redundáns a 15. után....

gomolygo committed:
Update 04-hosszukard.md

- előrehaladás -> sima keresztlépéses előre haladás

gomolygo committed:
Update 04-hosszukard.md

sok sok typo javítás

gomolygo committed:
Update 04-hosszukard.md

- támadás elkerülése
